WeakAuras/Prototypes.lua

```diff
@@ -867,7 +867,9 @@ end
 
 function WeakAuras.CheckCombatLogFlags(flags, flagToCheck)
   if type(flags) ~= "number" then return end
-  if (flagToCheck == "InGroup") then
+  if(flagToCheck == "Mine") then
+    return bit.band(flags, 1) == 1;
```

Comment:
Arguably we should use the COMBATLOG_OBJECT_MASK globals (or whatever they're named) but I don't really care that much.